<ui version="4.0" >
 <class>icqSettingsClass</class>
 <widget class="QWidget" name="icqSettingsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>icqSettings</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>411</width>
        <height>306</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Main</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="../../../../qutim.qrc" >
        <normaloff>:/icons/crystal_project/settings.png</normaloff>:/icons/crystal_project/settings.png</iconset>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="QFrame" name="frame" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="_2" >
          <property name="verticalSpacing" >
           <number>4</number>
          </property>
          <item row="0" column="0" >
           <widget class="QCheckBox" name="autoBox" >
            <property name="text" >
             <string>Autoconnect on start</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3" >
           <widget class="QCheckBox" name="saveStatusOnExitBox" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Save my status on exit</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="4" >
           <widget class="QCheckBox" name="avatarBox" >
            <property name="text" >
             <string>Don't send requests for avatarts</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="4" >
           <widget class="QCheckBox" name="reconnectBox" >
            <property name="text" >
             <string>Reconnect after disconnect</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="clientLabel" >
            <property name="text" >
             <string>Client ID:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" >
           <widget class="QComboBox" name="clientComboBox" >
            <item>
             <property name="text" >
              <string>qutIM</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>ICQ 6</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>ICQ 5.1</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>ICQ 5</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>ICQ Lite 4</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>ICQ 2003b Pro</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>ICQ 2002/2003a</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Mac ICQ</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>QIP 2005</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>QIP Infium</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>-</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1" >
           <widget class="QLabel" name="protoLabel" >
            <property name="text" >
             <string>Protocol version:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2" >
           <widget class="QSpinBox" name="protocolNumBox" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="4" column="3" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>31</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="0" colspan="2" >
           <widget class="QLabel" name="capabLabel" >
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text" >
             <string>Client capability list:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2" >
           <widget class="QLineEdit" name="capEdit1" >
            <property name="maxLength" >
             <number>32</number>
            </property>
           </widget>
          </item>
          <item row="6" column="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="0" colspan="2" >
           <widget class="QLineEdit" name="capEdit2" >
            <property name="maxLength" >
             <number>32</number>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="2" >
           <widget class="QLineEdit" name="capEdit3" >
            <property name="maxLength" >
             <number>32</number>
            </property>
           </widget>
          </item>
          <item row="9" column="0" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
         <zorder>autoBox</zorder>
         <zorder>saveStatusOnExitBox</zorder>
         <zorder>avatarBox</zorder>
         <zorder>reconnectBox</zorder>
         <zorder>clientLabel</zorder>
         <zorder>clientComboBox</zorder>
         <zorder>protoLabel</zorder>
         <zorder>protocolNumBox</zorder>
         <zorder>capabLabel</zorder>
         <zorder>capEdit1</zorder>
         <zorder>capEdit2</zorder>
         <zorder>capEdit3</zorder>
         <zorder></zorder>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>411</width>
        <height>306</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Advanced</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="../../../../qutim.qrc" >
        <normaloff>:/icons/crystal_project/advanced.png</normaloff>:/icons/crystal_project/advanced.png</iconset>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Account button and tray icon</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" >
          <item row="0" column="0" >
           <layout class="QVBoxLayout" name="verticalLayout" >
            <item>
             <widget class="QRadioButton" name="mainIconRadio" >
              <property name="text" >
               <string>Show main status icon</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="xIconRadio" >
              <property name="text" >
               <string>Show custom status icon</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="lastIconRadio" >
              <property name="text" >
               <string>Show last choosen</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>155</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="../../../../qutim.qrc" />
 </resources>
 <connections/>
</ui>
